pkg/osquery/table/platform_tables_darwin.go

```diff
@@ -13,19 +13,62 @@ import (
 	"github.com/kolide/launcher/pkg/osquery/tables/ioreg"
 	"github.com/kolide/launcher/pkg/osquery/tables/mdmclient"
 	"github.com/kolide/launcher/pkg/osquery/tables/munki"
+	"github.com/kolide/launcher/pkg/osquery/tables/osquery_user_exec_table"
 	"github.com/kolide/launcher/pkg/osquery/tables/profiles"
 	"github.com/kolide/launcher/pkg/osquery/tables/pwpolicy"
-	"github.com/kolide/launcher/pkg/osquery/tables/screenlock"
 	"github.com/kolide/launcher/pkg/osquery/tables/systemprofiler"
 	osquery "github.com/kolide/osquery-go"
 	"github.com/kolide/osquery-go/plugin/table"
 	_ "github.com/mattn/go-sqlite3"
 )
 
+const (
+	keychainAclsQuery  = "select * from keychain_acls"
+	keychainItemsQuery = "select * from keychain_items"
+	screenlockQuery    = "select enabled, grace_period from screenlock"
+)
+
 func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, currentOsquerydBinaryPath string) []*table.Plugin {
 	munki := munki.New()
 
+	// This table uses undocumented APIs, There is some discussion at the
+	// PR adding the table. See
+	// https://github.com/osquery/osquery/pull/6243
+	screenlockTable := osquery_user_exec_table.TablePlugin(
+		client, logger, "kolide_screenlock",
+		currentOsquerydBinaryPath, screenlockQuery,
+		[]table.ColumnDefinition{
+			table.IntegerColumn("enabled"),
+			table.IntegerColumn("grace_period"),
+		})
+
+	keychainAclsTable := osquery_user_exec_table.TablePlugin(
+		client, logger, "kolide_keychain_acls",
+		currentOsquerydBinaryPath, keychainItemsQuery,
+		[]table.ColumnDefinition{
```

Comment:
I can probably parse these...

Reply:
I wrote parsing code, merged it, and then rolled it back. It felt too complicated and not worthwhile.

Reply:
But checkout https://github.com/kolide/launcher/pull/741/commits/05abdd712cc99b1a8185ff41a8aac69a8251c0e9 if future me is curious